portal_pages/models.py

```diff
@@ -428,3 +433,90 @@ class OrganizationCaseStudy(Orderable, models.Model):
     panels = [
         FieldPanel('organization'),
     ]
+
+
+# Blog index page
+
+
+class BlogIndexPage(Page):
```

Comment:
Looks like TopImage is already in this branch, not sure which version this was...